source/vibe/inet/path.d

```diff
@@ -94,17 +94,17 @@ struct Path {
 	}
 	
 	/// Converts the Path object to a native path string (backslash as path separator on Windows).
-	string toNativeString()
+	string toNativeString() nothrow
 	const {
 		Appender!string ret;
 		
 		// for absolute unix paths start with /
-		version(Posix) { if(absolute) ret.put('/'); }
+		version(Posix) { if (m_absolute) ret.put('/'); }
 		
 		foreach( i, f; m_nodes ){
 			version(Windows) { if( i > 0 ) ret.put('\\'); }
 			version(Posix) { if( i > 0 ) ret.put('/'); }
-			else { enforce("Unsupported OS"); }
+			else { static assert(0, "Unsupported OS"); }
```

Comment:
Oh damn, I just noticed now how messed up that `enforce` call was...
```
version(Windows) { if( i > 0 ) ret.put('\\'); }
else version(Posix) { if( i > 0 ) ret.put('/'); }
else static assert(false, "Unsupported OS");
```

should be fine, though.

BTW, looks like a good warning opportunity for DMD.